stable/grafana/templates/job.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
       - name: {{ template "grafana.server.fullname" . }}-set-datasource
-        image: "{{ .Values.server.setDatasource.image }}"
+        image: "{{ default "appropriate/curl:latest" .Values.server.setDatasource.image }}"
```

Comment:
We recommend having the default values in the values.yaml. Please revert this change and uncomment the default value in the values.yaml file.